terminator/terminator_simple.py

```diff
@@ -37,5 +37,9 @@ def objective(trial):
     return scores.mean()
 
 
-study = optuna.create_study(direction="maximize")
-study.optimize(objective, callbacks=[TerminatorCallback()])
+if __name__ == "__main__":
+    study = optuna.create_study(direction="maximize")
+    study.optimize(objective, callbacks=[TerminatorCallback()])
```

Comment:
```suggestion
    study.optimize(objective, n_trials=50, callbacks=[TerminatorCallback()])
```
I've realized that specifying `n_trials` is better in cases where the trials are not terminated.